votor/src/event_handler.rs

```diff
@@ -437,6 +480,7 @@ impl EventHandler {
             slot,
             &voting_context.commitment_sender,
         )?;
+        stats.commitment_updates = stats.commitment_updates.saturating_add(1);
```

Comment:
This doesn't seem that useful given we're already reporting the number of each of type of vote